boards/riscv/tl321x/Kconfig.defconfig

```diff
@@ -0,0 +1,65 @@
+# Copyright (c) 2023 Telink Semiconductor
+# SPDX-License-Identifier: Apache-2.0
+
+if BOARD_TL3218X
+
+config BOARD
+	default "tl3218x"
+
+endif
+
+if BOARD_TL3218X_RETENTION
+
+config BOARD
+	default "tl3218x_retention"
+
+endif
+
+config HEAP_MEM_POOL_SIZE
+	default 4096
+
+if BOARD_TL3218X || BOARD_TL3218X_RETENTION
+
+config SOC_FLASH_TELINK_TLX
+	default y if FLASH
+
+config USB_TELINK_TLX
+	default y
+
+if BT
+
+# BLE Controller SDK from hal_telink requires
+# Telink's toolchain with FPU support
+config FPU
+	default y if "$(ZEPHYR_TOOLCHAIN_VARIANT)" != "zephyr"
+
+config BT_HCI_ACL_FLOW_CONTROL
+	default n
+
+choice BT_HCI_BUS_TYPE
+	default BT_TLX
+endchoice
+
+endif # BT
+
+# Workaround for not being able to have commas in macro arguments
+DT_CHOSEN_Z_CODE_PARTITION := zephyr,code-partition
+
+config FLASH_LOAD_OFFSET
+	default $(dt_chosen_reg_addr_hex,$(DT_CHOSEN_Z_CODE_PARTITION)) if USE_DT_CODE_PARTITION
+
+# Buffer for image writter shall be less(multiple of access alignment) or
+# equal to flash page. tlsr9518adk80d boards use external P25Q16 IC as
```

Comment:
Find globally in all new files keys like
b9
tlsr9
And replace or remove them

Reply:
Understood. Will replace them one by one later.